l10n_ch_import_camt054/models/account_bank_statement_line.py

```diff
@@ -41,11 +41,14 @@ def camt054_reconcile(self, account_code):
 
         # Group each line by acct_svcr_ref
         for acct_svcr_ref in set(all_move_lines.mapped('acct_svcr_ref')):
-            move_lines = move_line_obj.search([
-                ('reconciled', '=', False),
-                ('account_id.code', '=', account_code),
-                ('acct_svcr_ref', '=', acct_svcr_ref)
-            ])
+            move_lines = all_move_lines.filtered(
+                lambda x: x.acct_svcr_ref == acct_svcr_ref)
             if len(move_lines) > 1 and sum(move_lines.mapped('debit')) == \
                     sum(move_lines.mapped('credit')):
-                move_lines.reconcile()
+                # the reconcile function makes use of the recursive
+                # auto_reconcile function. This function is slow due to calls
+                # to _compute_partial_lines. This was improved in Odoo v12
+                # (see d3d26120614139fd7d7e888bd66d21de5158a034).
+                # We therefore skip the call when move_lines is too big.
+                if float(release.version) > 12 or len(move_lines) < 500:
```

Comment:
No chance that release.version will be equal to 12? Or would `>=` be a better operator?